ethereum_history_api/circuits/lib/src/verifiers/receipt_test.nr

```diff
@@ -41,3 +44,46 @@ fn test_assert_tx_idx_equals_fail() {
     let wrong_key = [0x8, 0x2, 0xa, 0xb, 0xc, 0xd + 1];
     assert_tx_idx_equals(wrong_key, tx_idx);
 }
+
+#[test]
+fn test_assert_receipt_proof() {
+    assert_receipt_proof(proof, receipts_root);
+}
```

Comment:
Should we have success test cases for more than different type of receipts?

Reply:
At the moment we have only one transaction and receipt fixture. Added [task](https://www.notion.so/vlayer/Tx-Receipt-Add-more-transaction-and-receipt-fixtures-and-update-test-fb0d48a0e94b47c384c12c58aa1dcd1a?pvs=4) to backlog:

Reply:
Now we have more then one receipt fixture so I added one more test